b2share/modules/schemas/serializers.py

```diff
@@ -110,3 +134,86 @@ def community_schema_to_json_serializer(community_schema, code=200,
     if headers is not None:
         response.headers.extend(headers)
     return response
+
+
+def new_schema_to_json_serializer(schema, code=200, headers=None):
```

Comment:
Why is it "new" schema and not "block" schema. Is used only for newly created schemas?